### DISCUSSION TITLE
All entries in collections are lost. How can I restore them?

### DISCUSSION THREAD

**Discussion by AlexTechNoir** [strapi/strapi#7697] [Legacy Topics]
Hello, everyone!

I was trying to deploy Strapi on Heroku. Since Heroku is incompatible with SQLite, I was following [this](https://youtu.be/HHnRsZwFmR8) video about "migrating" Strapi to MongoDB Atlas.

Everything was going well, but. After I `cross-env NODE_ENV=production npm start` I couldn't log in Strapi account, because log in page was redirecting me to register page. I saw a couple of issues about this on GitHub, but none of them helped me. 

Not knowing what to do I attempted to enter my credentials in register form, hoping that system will recognize me. And it did! But when I entered into the account all my entries were gone, however the collections are intact.

![0](https://user-images.githubusercontent.com/43852391/91956419-4b2fc100-ed0d-11ea-8564-65198b406edb.jpg)

Also, all images that I've uploaded are lost too:

![1](https://user-images.githubusercontent.com/43852391/91957363-96969f00-ed0e-11ea-8b04-7e2e6ace9ead.jpg)
![2](https://user-images.githubusercontent.com/43852391/91957990-6d2a4300-ed0f-11ea-9a58-45108ba1166a.jpg)

How can I restore entries and images?

**Comment by derrickmehaffy** [comment]
You will have to manually migrate these, we don't currently provide an option to migrate your data from SQLite to MongoDB (or any database to any other).

See this troubleshooting FAQ: https://strapi.io/documentation/v3.x/getting-started/troubleshooting.html#does-strapi-handle-deploying-or-migrating-of-content

You will need to do this manually due to the differences in the ID type, migrating to PostgreSQL will be easier since they both use integers as the ID type but this will still be a manual process. You can use a local DB Client like DBeaver or HeidiSQL to load the SQLite file and export the data into a standard format, but keep in mind that if you plan to go to MongoDB all of your relations will be broken.

**Reply by AlexTechNoir** [reply]
Thank you, Derrick!

I'll get back to SQLite, create a new project with Mongo and manually copy the content, then.
